packages/dds/cell/src/test/cell.spec.ts

```diff
@@ -161,55 +161,55 @@ describe("Cell", () => {
         });
 
         describe("Garbage Collection", () => {
-            let subCellCount = 0;
-            let expectedRoutes: string[] = [];
-
-            beforeEach(() => {
-                subCellCount = 0;
-                expectedRoutes = [];
-            });
-
-            // Return the remote SharedCell because we want to verify its summary data.
-            const getSharedObject = () => cell2;
-
-            async function addOutboundRoutes() {
-                const factory = new CellFactory();
-                const newSubCell = factory.create(dataStoreRuntime, `subCell-${++subCellCount}`);
-                cell.set(newSubCell.handle);
-                expectedRoutes = [ newSubCell.handle.absolutePath ];
-                containerRuntimeFactory.processAllMessages();
-            }
-
-            async function deleteOutboundRoutes() {
-                cell.delete();
-                expectedRoutes = [];
-                containerRuntimeFactory.processAllMessages();
+            class GCSharedCellProvider implements IGCTestProvider {
+                private subCellCount = 0;
+                private _expectedRoutes: string[] = [];
+
+                constructor() {
+                    this.subCellCount = 0;
```

Comment:
constructor is not needed - you already init variables where they are declared

Reply:
oh yeah, this is class now!